.naverc

```diff
@@ -0,0 +1 @@
+lts/carbon
```

Comment:
I find it better to put a specific version as the names are somewhat fluid

Reply:
`lts/carbon` == `8.x.x`. I'm gonna leave it as this matches what we have in Backpack.

Reply:
The motivation behind relying on the aliases is that they are intended to be stable with only bug fixes and security patches. And FYI the current LTS is `lts/dubnium` ...

Reply:
Ah yes, Dubnium, the element from which we produce dubstep music

Reply:
Ok cool 👍